cmd/scollector/collectors/interval.go

```diff
@@ -61,8 +63,11 @@ func (c *IntervalCollector) Run(dpchan chan<- *opentsdb.DataPoint) {
 				slog.Errorf("%v: %v", c.Name(), err)
 				result = 1
 			}
-			Add(&md, "scollector.collector.duration", timeFinish.Seconds(), tags, metadata.Gauge, metadata.Second, "Duration in seconds for each collector run.")
-			Add(&md, "scollector.collector.error", result, tags, metadata.Gauge, metadata.Ok, "Status of collector run. 1=Error, 0=Success.")
+			if !DisableSelf {
```

Comment:
What about using `collect.DisableDefaultCollectors` instead of adding a new variable? Your choice, just a thought. Otherwise LGTM.